packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -267,7 +274,7 @@ ts_library(
             "src/**/tests/*.ts",
         ],
         exclude = [
-            "src/testing/**/*_spec.ts",
+            "src/**/*_spec.ts",
```

Comment:
It's actually necessary because otherwise the Jest test sources get pulled into multiple `ts_library()` targets and conflict with each other during analysis.